napari/_tests/test_with_screenshot.py

```diff
@@ -209,7 +209,7 @@ def test_changing_image_gamma(make_napari_viewer):
 
     screenshot = viewer.screenshot(canvas_only=True, flash=False)
     center = tuple(np.round(np.divide(screenshot.shape[:2], 2)).astype(int))
-    assert screenshot[center + (0,)] == 128
```

Comment:
This test unfortunately is different; after a chat with @tlambert03, we concluded that this should be fine a bit more lax.